src/conv/k-g/k_parser.cpp

```diff
@@ -267,7 +394,16 @@ bool parse_k
 			break;
 
 		    case KState::Node: {
-			tokens = read_line_node_standard(line.c_str());
+			if (nodeFormat == ReadFormat::Standard) {
+			    tokens = read_line_node_standard(line.c_str());
+			}
+			else if (nodeFormat == ReadFormat::Long) {

```

Comment:
You need to catch the two line format here. E.g.:
* have a variable for the first line
* if this variable is empty and the result of read_line_node_long() has the size 4 => write it to the variable and go on
* if this variable is not empty => add the result of read_line_node_long() to it, generate the node, and clear the variable

Reply:
yes, thank you for the notes.